csc-overrides/partials/disclaimer.html

```diff
@@ -0,0 +1,91 @@
+{% from "partials/commit.html" import commit_link %}
+<div id="disclaimer-container">
+  <div class="disclaimer preview">
+    <p class="disclaimer-title">You are viewing a developmental build</p>
```

Comment:
```suggestion
    <p class="disclaimer-title">You are viewing a development build</p>
```

It sounds nicer to me, what do you think?